flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -831,6 +888,96 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.inOut = data;
     }
 
+    clearFocus() {
+        this.inFocus = false;
+        Coherent.trigger('UNFOCUS_INPUT_FIELD');
+        this._inOutElement.style = null;
+        this.getChildById("header").style = null;
+        if (this.check_focus) {
+            clearInterval(this.check_focus);
+        }
+    }
+
+    initKeyboardScratchpad() {
+        window.document.addEventListener('click', () => {
+
+            const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "DISABLED");
```

Comment:
use `this.isInteractive()` instead OR scrap the method.

Reply:
See above comment on get isInteractive()